portfolio/src/main/webapp/index.html

```diff
@@ -10,5 +10,59 @@
     <script src="script.js" defer></script>
   </head>
   <body>
+    <!--NAV BAR-->
+    <nav>
+      <ul>
+        <li><a href="#home">Home</a></li>
+        <li><a href="#work">Work</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <!--HOME-->
+    <section id="home">
+      <h1 id="greeting">Hi there!</h1>
+      <p>I am Lin Wei.</p>
+      <!--TODO: include action verbs and animate it using js-->
+      <p>I <span id="actions"></span></p>
+    </section>
+
+    <!--FEATURED WORKS-->
+    <section id="works">
+      <div class="title">
+        <h1>Featured Works</h1>
+      </div>
+      <!--CONTAINER FOR ALL PROJECTS-->
```

Comment:
No need to keep a newline for this. If there's more work to be done, add a todo instead of leaving space/temp text for future work.

E.g, for here you can just do 
```
<!--TODO: Add all projects to container-->
<div id="projects"/>
```

Reply:
I see. Okay! Noted with thanks :D

Reply:
Also, feel free to resolve the comment if you believe that they're resolved :)

Reply:
Okay!